packages/utils/telemetry-utils/src/index.ts

```diff
@@ -61,13 +40,106 @@ export { MockLogger } from "./mockLogger";
 export { ThresholdCounter } from "./thresholdCounter";
 export { SampledTelemetryHelper } from "./sampledTelemetryHelper";
 export { logIfFalse } from "./utils";
+export { ITelemetryEventExt } from "./telemetryTypes";
+
+// Deprecated exports (moved to core-interfaces).
+// Kept here to preserve compatibility. They will be removed in a subsequent release.
+
 export {
+	/**
+	 * @deprecated Import from "\@fluidframework/core-interfaces" instead.
+	 */
```

Comment:
does this work? it doesn't show up in the api-docs? does intellisense support it?

Reply:
Intellisense supports it! I was pleasantly surprised.